changelogs/fragments/6719-redfish-utils-fix-for-get-volume-inventory.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_utils - fix for ``GetVolumeInventory`` (https://github.com/ansible-collections/community.general/pull/6719).
```

Please also update the text itself, a little bit more details would be helpful.
```suggestion
  - redfish_utils module utils - fix for ``GetVolumeInventory`` (https://github.com/ansible-collections/community.general/pull/6719).
```